### PR TITLE
fix(mcdu): change page title of init b

### PR DESCRIPTION
**Pull request by Revyn112**
Fixes #[issue_no]

## Summary of Changes
Based on observations of some A320neo cockpit documentations, reading the FCOM and the Honeywell Pegasus the INIT B page title is change dynamically based on the fuel planning state.

## Additional context

Discord username (if different from GitHub):

## Testing instructions

1. Setup INIT A page
2. Switch to INIT FUEL PRED page.
3. Add ZFW/ZFWCG
4. Press LSK3R to start FUEL PLANNING
5. Page title should change to "INIT FUEL PLANNING"
6. Finish FUEL PLANNING and press LSK3R "BLOCK CONFIRM"
6. INIT B page title should now be and stay "INIT FUEL PRED"

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by BlueberryKing** [comment]
Hi, I think some of your references do not apply to the FMS version that we are modeling (Honeywell H3). The "INIT B" page has been renamed to the "INIT FUEL PRED" page. I believe this will also be the page title before the engines are started, regardless of whether a block fuel has been entered or not.

I think the page title changing to INIT FUEL PLANNING when that function is used is correct though.

**Comment by pinatacolada** [comment]
the behavior you are trying to implement is of Honeywell H2C and Thales S7B and beyond.

The FBW uses HW H3, on which the page is always called `INIT FUEL PRED`, except if FUEL PLANNING was selected, where it will be renamed to `INIT FUEL PLANNING`. It then returns to the original title when FOB is confirmed

File: fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_InitPage.js [hunk 1]
```diff
@@ -413,6 +413,7 @@ class CDUInitPage {
 
         };
 
+        let fuelPageTitle = new Column(10, "INIT");
```
Comment (pinatacolada) [inline fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_InitPage.js:416]:
```suggestion
        let fuelPageTitle = new Column(10, "INIT FUEL PRED");
```

File: fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_InitPage.js [hunk 2]
```diff
@@ -532,6 +534,7 @@ class CDUInitPage {
         }
 
         if (CDUInitPage.fuelPredConditionsMet(mcdu)) {
+            fuelPageTitle = new Column(1, "INIT FUEL PREDICTION");
```
Comment (pinatacolada) [inline fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_InitPage.js:537]:
```suggestion
            fuelPageTitle = new Column(1, "INIT FUEL PRED");
```

**Review by 2hwk — APPROVED** [review]

**Comment by tracernz** [comment]
We are awaiting a pilot to check this one, as it doesn't match the new diagram but the text (same as the H2c text) does say the title changes.

**Comment by 2hwk** [comment]
Also testing instructions need to be updated (most likely), since the code seems to follow pina's comment now

**Comment by Revyn112** [comment]
Updated testing instructions

**Comment by brewers10** [comment]
Quality Assurance Trainee Report

Discord : brewers_10#5706
Object of testing: #7891 
Tier of Testing: 2
Date: 02/04/2023

Testing Process: 
Followed process step by step. I've attached images for each step in the media section as it's easier than explaining. 

Negatives:       
N/A
Testing Results: 
Passed

Conclusions: 
LGTM

Media:       
Microsoft Flight Simulator Screenshot 2023 04 02 - 13 31 08 05
Microsoft Flight Simulator Screenshot 2023 04 02 - 13 31 29 77
Microsoft Flight Simulator Screenshot 2023 04 02 - 13 29 30 19
Microsoft Flight Simulator Screenshot 2023 04 02 - 13 30 02 71
Microsoft Flight Simulator Screenshot 2023 04 02 - 13 31 01 46

**Comment by 2hwk** [comment]
Just waiting on accuracy check, moved to awaiting action

**Comment by BlueberryKing** [comment]
Hi @Revyn112, according to a pilot, the "BLOCK CONFIRM" text should also be in amber with an asterisk * next to it.

**Comment by tracernz** [comment]
Turns out the text is out of date while the new illustrations are correct and thus we already have it correct. We can close this. 

@BlueberryKing that does not appear to be the case in our photographs.

**Comment by BlueberryKing** [comment]
> Turns out the FCOM text is out of date while the new illustrations are correct and thus we already have it correct. We can close this.

I agree with the first part, but I don't think we have it quite correct. For example, this state should not exist IRL.
It should either show FUEL PLANNING in green, or BLOCK CONFIRM in cyan with an asterisk.

block-confirm-before-fix

**Comment by tracernz** [comment]
Different issue not related to this PR.

**Comment by tracernz** [comment]
We have confirmed that the existing behaviour is correct so this PR is not needed.

**Pull request closed without merging**